OfficeAddin/05 Using modern JavaScript/Demos/01 Office Add-in using React/README.md

```diff
@@ -190,524 +128,574 @@ In this section, you will finish developing the Office Add-in using React and Ty
-                    var range = sheet.getRange(address);
-                    range.values = [[value]];
-                    return context.sync().then(async () => {
-                        resolve();
-                    });
-                }).catch((err) => {
-                    reject(err);
+          );
+        }
+        ```
+
+    >Note: This is a good time to test the **delete symbol** function of your add-in.
+
+1. Update the **ExcelTableUtil** utility to add support for refreshing rows in the table:
+    1. Locate and open the **src/components/ExcelTableUtil.tsx** file.
+    1. Add the following methods to the `ExcelTableUtil` class:
+
+        ```typescript
+        // Updates a specific cell in the table
+        updateCell = async (address: string, value: any) => {
+          return new Promise(async (resolve, reject) => {
+            this.ensureTable(true).then(
+              async () => {
+                await Excel.run(async context => {
+                  const sheet = context.workbook.worksheets.getActiveWorksheet();
+                  const range = sheet.getRange(address);
+                  range.values = [[value]];
+                  return context.sync().then(async () => {
+                    resolve();
+                  });
+                }).catch(err => {
+                  reject(err);
                 });
-            }, (err) => {
+              },
+              err => {
                 reject(err);
-            });
-        });
-    }
-    ````
-
-1. Next, update the **refreshSymbol** function on **src/components/app.tsx** to call **getQuote** for updated stock statistics and then update the last trade cell in the Excel table. Similar to deleteSymbol, this function will call **getColumnData** to determine the cell address before calling **updateCell**.
-
-    ````typescript
-    // Refresh symbol
-    refreshSymbol = async (index:number) => {
-        // Refresh stock quote and update Excel table
-        let symbols = this.state.listItems;
-        let symbol = symbols[index];
-        this.setState({ waiting: true });
-        this.tableUtil.getColumnData("Symbol").then(async (columnData:string[]) => {
-            // make sure the symbol was found in the Excel table
-            var rowIndex = columnData.indexOf(symbol);
-            if (rowIndex != -1) {
-                this.getQuote(symbol).then((res:any) => {
-                    // "last trade" is in column B with a row index offset of 2 (row 0 + the header row)
-                    this.tableUtil.updateCell(`B${rowIndex + 2}:B${rowIndex + 2}`, res.current).then(async () => {
+              }
+            );
+          });
+        }
+        ```
+
+1. Update the **App** component to leverage the methods you added to the `ExcelTableUtil` class.
+    1. Locate and open the **src/components/App.tsx** file.
+    1. Locate and update the `refreshSymbol()` method in the `App` class to refresh specifying symbol in the Excel table
```

Comment:
Locate and update the `refreshSymbol()` method in the `App` class to specify a symbol to refresh in the Excel table?